packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -26,55 +28,77 @@ class DebuggerScreen extends Screen {
   }
 }
 
-class DebuggerScreenBody extends StatelessWidget {
+class DebuggerScreenBody extends StatefulWidget {
   @override
-  Widget build(BuildContext context) {
-    if (!kIsWeb) {
-      final theme = Theme.of(context);
-      final textStyle = Theme.of(context)
-          .textTheme
-          .headline5
-          .copyWith(color: theme.accentColor);
-      return Center(
-        child: Text(
-          'The debugger screen is only available when running DevTools as a web'
-          'app.\n'
-          '\n'
-          'It is implemented as a webview, which is not available in Flutter '
-          'desktop embedding.',
-          style: textStyle,
-          textAlign: TextAlign.center,
-        ),
-      );
-    }
+  DebuggerScreenBodyState createState() => DebuggerScreenBodyState();
+}
 
-    // TODO(https://github.com/flutter/flutter/issues/43532): Don't build const
-    // because compile time const evaluation will fail on non-web apps.
-    // ignore:prefer_const_constructors
-    final webView = HtmlElementView(
-      viewType: 'DebuggerFlutterPlugin',
-    );
+class DebuggerScreenBodyState extends State<DebuggerScreenBody> {
+  DebuggerState debuggerState;
+  Script script;
 
-    // Wrap the content with an EagerGestureRecognizer to pass all mouse
-    // events to the web view.
-    return RawGestureDetector(
-      gestures: {
-        EagerGestureRecognizer: _EagerGestureFactory(PointerDeviceKind.mouse),
-      },
-      child: webView,
+  @override
```

Comment:
add a TODO to add a dispose and handle the lifecycle  so we avoid the setState after dispose for the case of loading a script slowly.

Reply:
Done.